main.cpp

```diff
@@ -125,419 +152,429 @@ struct CarWash
- */
+
+void Band::playForAudience(char whichSong, int tempo, bool countOff)
+{
+    if(countOff)
+    {
+        tempo = 130;
+    }
+    whichSong = 'V';
+}
+std::string Band::nameASong(std::string word1, std::string word2)
+{   
+    std::string songTitle = word1 + word2;
+    return songTitle;
+}
+
+void Band::postOnInsta(int numOfPics, char filter, std::string postText)
+{
+    numOfPics = 5;
+    filter = 'N';
+    postText = "Had an awesome time at Club Buddhabump last nite!";
+}
+
+
+
 struct EspressoMachine
 {
-    //Amount of water in tank
     float cupsOfWater;
-    //Type of coffee
     char coffeeType;
-    //water temperature
     int waterTempInF;
-    // tablespoons of coffee grounds
     float tbspOfGrounds;
-    //steamer pressure
     int steamerPressure;
 
-    //Heat water
     void heatWater(int waterLevel, int targetTempInF);
-    //Pour coffee
     void pourCoffee(float pourTime);
-    //Make sound when done
     void makeSoundWhenDone(int loudness);
 };
-/*
-Thing 4) Airplane
-5 properties:
-    1) Crew members
-    2) Passenger members
-    3) Capacity
-    4) Engine power
-    5) total weight
-3 things it can do:
-    1) retract/extend landing gear
-    2) increase/decrease engine power
-    3) Angle wings up/down
- */
+
+void EspressoMachine::heatWater(int waterLevel, int targetTempInF)
+{
+    if(waterTempInF < targetTempInF)
+    {
+        waterTempInF += 10/waterLevel;
+    }
+}
+void EspressoMachine::pourCoffee(float pourTime)
+{
+    for(int i = 0; i < pourTime ; i++)
+    {
+        cupsOfWater -= 1;
+    }
+}
+void EspressoMachine::makeSoundWhenDone(int loudness)
+{
+    loudness *= 2;
+}
+
+
 struct Airplane
 {
-    //num of Crew members
     int numOfCrew;
-    //num of Passenger members
     int numOfPassengers;
-    //maximumCapacity
     int maxCapacity;
-    // engine horse power
     float engineHorsePower;
-    //steamer pressure
     int totalWeightInLbs;
 
-    //extend lending gear
     void extendLandingGear(bool landingGearExtended);
-    //decrease engine power
     void changeEnginePower(float amountChanged, int whichEngine);
-    //Angle wings up
-    int angleWingsForTakeoff();
+    bool angleWingsForTakeoff(int wingAngleDegree);
 };
-/*
-Thing 5) Oscillator
-5 properties:
-    1) Tuning (semitones)
-    2) Waveform
-    3) Phase (degree)
-    4) Noise level
-    5) LFO amount
-3 things it can do:
-    1) Make signal
-    2) Sync with another oscillator
-    3) Modulate other parameter
- */
+
+void Airplane::extendLandingGear(bool landingGearExtended)
+{
+    if(landingGearExtended == false)
+    {
+        landingGearExtended = true;
+    }
+}
+void Airplane::changeEnginePower(float amountChanged, int whichEngine)
+{
+    engineHorsePower += amountChanged;
+    whichEngine = 2;
+}
+bool Airplane::angleWingsForTakeoff(int wingAngleDegree)
+{
+    wingAngleDegree += 3;
+    bool isFlying = false;
+    if( engineHorsePower > 2000)
+    {
+        isFlying = true;
+    }
+    return isFlying;
```

Comment:
you can write `return engineHorsePower > 2000;` instead of writing this `if()` block
- [ ] revise